src/Catalyst.Core.Lib/Config/ConfigCopier.cs

```diff
@@ -44,10 +44,7 @@ public abstract class ConfigCopier : IConfigCopier
             Guard.Argument(dataDir, nameof(dataDir)).NotNull().NotEmpty().NotWhiteSpace();
 
             var dataDirInfo = new DirectoryInfo(dataDir);
-            if (!dataDirInfo.Exists)
-            {
-                dataDirInfo.Create();
-            }
+            if (!dataDirInfo.Exists) dataDirInfo.Create();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)